tests/test_parseException.py

```diff
@@ -23,8 +23,8 @@
 
 from api.globalConfig import yc_gc
 from parseAndPopulate.loadJsonFiles import LoadFiles
-from parseAndPopulate.modules import Modules
-from parseAndPopulate.parseException import ParseException
+from parseAndPopulate.modules import SdoModule
+from utility.yangParser import ParseException
```

Comment:
If `ParseException` was moved to `utility.yangParser` maybe this test Class should be moved into utility/tests/test_parseException.py ??